Sources/MapboxDirections/DirectionsResult.swift

```diff
@@ -139,6 +143,11 @@ open class DirectionsResult: Codable {
      */
     open var expectedTravelTime: TimeInterval
     
+    /**
+     The route’s typical travel time, measured in seconds.
+     */
+    open var typicalTravelTime: TimeInterval?
```

Comment:
This property is on DirectionsResult rather than Route, so it’s also available on Match. Does the Map Matching API expose this value as well?

Reply:
Just did a few tests related to Map Matching API and don't see that `duration_typical` is exposed.